app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java

```diff
@@ -304,6 +306,18 @@ private void loadAbout(final WebView webView) {
         final String wordmark = HtmlLoader.loadPngAsDataURI(webView.getContext(), R.drawable.wordmark);
         substitutionMap.put("%wordmark%", wordmark);
 
+        String direction = "auto";
```

Comment:
nit: If you do not set "`auto`" here then you can define `direction` as `final` and set `auto` from an '`else`' branch in your code below.